src/thrift_0_13_0/.gitignore

```diff
@@ -0,0 +1,3 @@
+!0001-Disable-unneeded-packages.patch
+!Makefile
+!series
```

Comment:
Seems this file is not necessary. They are not ignored by default.

Reply:
Still have this concern.

Reply:
Just follow the same approach as in src/thrift, but agree it is not the best.
Added here just one rule thrift*   to ignore downloaded archives and extracted files